src/components/MultiHopTrade/components/SharedTradeInput/SharedTradeInputFooter/SharedTradeInputFooter.tsx

```diff
@@ -123,7 +123,7 @@ export const SharedTradeInputFooter = ({
             isDisabled={shouldDisableGasRateRowClick}
             rate={rate}
             deltaPercentage={deltaPercentage?.toString()}
-            isLoading={isLoading}
+            isLoading={isLoading && !rate}
```

Comment:
Ensures no loading state so long as we have a rate, will only display `RateGasRow` as loading on initial fetch